### ISSUE TITLE
Neither C++ binaries compile

### ISSUE THREAD

**Issue by SimPilotAdamT** [ZhengC1/PenisParser#55]
Tried compiling both C++ versions 1 and 2, but neither compiles using the latest Ubuntu gcc (I also had the same issue with the gcc on Arch Linux, KDE Neon, and Manjaro Linux among others).

Shell output:
```Shell
[22:27:41] adam@Adam-ThinkPad-L380 /home/adam/PenisParser/c++ [2]  
> make v2
g++ -std=c++11 C++V2/penis.cpp -o bin/penis
/usr/bin/ld: cannot open output file bin/penis: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [makefile:10: v2] Error 1
[22:27:46] adam@Adam-ThinkPad-L380 /home/adam/PenisParser/c++ [2]  
> make v1
g++ -std=c++11 C++V1/penis.cpp -o bin/penis
/usr/bin/ld: cannot open output file bin/penis: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [makefile:7: v1] Error 1
[22:28:17] adam@Adam-ThinkPad-L380 /home/adam/PenisParser/c++ [2]  
> 
```

- [x] Redo the C++ makefile

I would do so myself and make a pull req, but I got exams atm ¯\\_(ツ)_/¯.

**Comment by MostHated** [comment]
Don't you need this *for* your exam? Don't tell me you plan on turning it in unparsed?

**Comment by SimPilotAdamT** [comment]
> Don't you need this *for* your exam?

For a *written* exam.... I already did my computer science exams last week, and I parsed those w the ones written in C. I decided just now what the hell and try to compile the C++ ones, but apparently it's not doable without fixing the makefile ;-;.

**Comment by SimPilotAdamT** [comment]
I fixed the makefile and made a pull request (#56). Going to close this issue as that pull req shouldn't be hard to do.
